bucket/geogebra.json

```
@@ -1,13 +1,25 @@
 {
     "version": "6.0.909.9",
     "description": "A dynamic mathematics software for education that brings together geometry, algebra, spreadsheets, graphing, statistics and calculus.",
-    "homepage": "https://www.geogebra.org/",
+    "homepage": "https://www.geogebra.org",
     "license": {
         "identifier": "Freeware",
         "url": "https://www.geogebra.org/license"
     },
+    "notes": [
+        "To register file associations, please execute the following command:",
+        "- `reg import \"$dir\\register-file-associations.reg\"`"
```

Even though [this has not been standardized yet](https://github.com/ScoopInstaller/Main/issues/7061), I would suggest using unified script names and removing unnecessary characters:
- `install-associations.reg`
- `uninstall-associations.reg`



```suggestion
        "reg import \"$dir\\install-associations.reg\""
```